Automattic/jetpack
# Premium Content Block: fix icon color

#### Changes proposed in this Pull Request:

Replaces `black` with `currentColor` in SVG icon. Allows icon to responsively update color depending on context.

I was originally going to use the `getIconColor` function but this has an adverse effect in the Add Block section.

#### Does this pull request change what data or activity we track or use?


Nope

#### Testing instructions:
* Create a new post
* Add the Premium Content block
* Open List view and click the Premium content block. Notice the icon matches the text color

<img width="812" alt="Markup 2021-10-07 at 12 49 34" src="https://user-images.githubusercontent.com/33258733/136370502-19f9fd0f-d2e9-4593-961f-8a9b7b96c92c.png">

Fixes Automattic/wp-calypso#48959
